index.html

```diff
@@ -41,43 +48,208 @@ <h1 class="about-title padding">I'm Rose Glad to see you!</h1>
+          <h1 class="about-title padding">About Myself</h1>
+        </div>
+        <div class="full-description">
+          <p class="about-me padding">Hello I’m a software developer! 
+            I can help you build a product , feature or website 
+            Look through some of my work and experience! If you like what you
+            see and have a project you need coded, don’t hestiate to contact me.
+          </p>
+        </div>
+        <div class="socials-section">
+          <button class="connect-link padding">LET'S CONNECT</button>
+          <ul class="icons">
+            <li><img src="./images/twitter.svg" class="icon" alt="twitter icon"></li>
+            <li><img src="./images/Linkedin.svg" class="icon" alt="twitter icon"></li>
+            <li><img src="./images/mail.svg" class="icon" alt="twitter icon"></li>
+            <li><img src="./images/github.svg" class="icon" alt="twitter icon"></li>
+            <li><img src="./images/playboy.svg" class="icon" alt="twitter icon"></li>
+          </ul>
+        </div>
+        <a href="#" class="see-project-btn">Get my resume</a>
+      </section>
+
+      <section class="languages">
+        <ul>
+          <li class="languages-title">Languages
+            <ul class="card-language">
+              <li class="js-img">
+                <img src="./images/Frame9.svg" alt="Javascript">
+              </li>
+                    
+              <li class="html-img">
+                <img src="./images/Frame10.svg" alt="HTML">
+              </li>
+                    
+              <li class="css-img">
+                <img src="./images/Frame11.svg" alt="CSS">
+              </li>
+            </ul>
+          </li>
+          <li class="framework">
+            <ul class="framework-list">
+              <li class="languages-title">Frameworks</li>
+              <img class="arrow-img" src="./images/Union.svg" alt="CSS">
+            </ul>
+          </li>
+          <hr>
+          <li class="skill">
+            <ul class="skill-list">
+              <li class="languages-title">Skills</li>
+              <img class="arrow-img" src="./images/Union.svg" alt="CSS">
+            </ul>
+          </li>
+          <hr>
+        </ul>
+
+      </section>
     </section>
-    
+   
   </main>
+
+  <!-- <footer class="footer">
+    <h2 class="contact-me-title">Contact me</h2>
+    <p class="contact-description">If you have an application you are interested in developing, 
+      a feature that you need built or a project that needs
+      coding. I’d love to help with it</p>
+    <form class="contact-form">
+      <input type="text" name="name" id="yourname" placeholder="Yeremias">
+      <input type="email" name="email" id="youremail">
+      <textarea name="message" id="yourmessage" cols="30" 
+        rows="10" placeholder="Write your essage here">
+      </textarea>
+      <button type="submit">Get in touch</button>
+    </form>
+
+  </footer> -->
```

Comment:
[OPTIONAL]
- [ ] It's not a good practice to leave commented lines of code in your code base, YOu may consider deleting them for clean code best practice.